src/index.ts

```diff
@@ -85,10 +96,7 @@ function validationRoute<
       // TODO: figure out which reporter we wanna use
       const report = PathReporter.report(result);
       if (result.isLeft()) {
-        return res.status(422).json({
-          success: false,
-          errors: report
-        });
+        throw new IoTsValidationError(JSON.stringify(report));
```

Comment:
Why stringify it?

Reply:
Message had to be a string..
but you mean I should rather stringify in the class?

Reply:
So perhaps the report can be added as a property on the error.
The message should prob be a more generic string. Perhaps a formatted version of the report. (I think iots has some built in report formatters)

Reply:
Their built in reporters are not that good.

Reply:
Agreed the report should be a property.